app/views/MainTempleteNew.scala.html

```diff
@@ -158,11 +159,11 @@
     }
 
     @attorneyBanner
-
     @backLinkUrl.map(url =>
-        govukBackLink(BackLink(attributes = Map("id" -> "returnToChooseTaxYearLink"), href = url, content = Text(backLinkContent)))
+        govukBackLink(BackLink(classes = "js-visible-back",
```

Comment:
Not sure if it makes a difference but I done this with Mas and he said to use "js-visible"?

Reply:
I rechecked and in the PROD the same is being used. Will recheck with Mas on return though.